platform/CMakeLists.txt

```diff
@@ -167,13 +169,17 @@ endif()
 
 if (PLATFORM_LINUX)
   set_target_properties(${PROJECT_NAME} PROPERTIES AUTOMOC ON)
```

Comment:
Should AUTOMOC be set on all desktop platforms?

Reply:
It was the `bsdiff_tests` which made it necessary here.
So far this PR was focusing on linux only,
perhaps as a preparation for the refactoring of the MacOS code, to leverage the `qt_location_service`,
Perhaps as such, it could make sense, but in general, IMHO all that would belong to a separate PR.